client/components/cart-summary.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import CartSummaryItems from './cart-summary-items';
+
+export default class CartSummary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.getTotal = this.getTotal.bind(this);
+  }
+
+  getTotal() {
+    let totalAmount = 0;
+    for (let i = 0; i < this.props.cartArray.length; i++) {
+      totalAmount += this.props.cartArray[i].price;
+    }
+    const totalString = totalAmount.toString();
+    if (totalAmount === 0) {
+      return '$0';
+    } else if (totalString.length === 3) {
+      return '$' + totalString[0] + '.' + totalString[1] + totalString[2];
+    } else if (totalString.length === 4) {
+      return '$' + totalString[0] + totalString[1] + '.' + totalString[2] + totalString[3];
+    } else if (totalString.length === 5) {
+      return '$' + totalString[0] + totalString[1] + totalString[2] + '.' + totalString[3] + totalString[4];
+    } else if (totalString.length === 6) {
+      return '$' + totalString[0] + totalString[1] + totalString[2] + totalString[3] + '.' + totalString[4] + totalString[5];
```

Comment:
This isn't really tenable. You can divide the total by 100 to get dollars and put the decimal in the right place with `.toFixed()`